src/common/mfu_flist_copy.c

```diff
@@ -381,33 +399,27 @@ static int mfu_copy_acls(
           * We use the opaque method as we are not trying to alter the ACL contents.
           * Note that if the source is not a GPFS file-system, then the call will
           * fail with EINVAL and so we never try to apply this to the dest_path */
-     
```

Comment:
Looks like about 5-6 empty lines got deleted.  Let's keep those in there since the spacing helps make it easier to read.